cfarm/api-1
# Ccdb api bug fixes

Copy + CSS + JS bug fixes for the CCDB api docs
## Additions
- cf-expandables css + js
- cf-icons + cf-typography css for the PDF icons on index page
- font files for the icons
- cf-tables css for the field reference table styling on small screens
## Removals
- outdated expandables js + css
## Changes
- changed "Release data" to "Release notes" in nav, filename, page title
- updated HTML for the Release notes expandables - lots o copy n paste so some extra eyes on this to make sure the content matches the previous version would be great.
## Testing

I tested the expandables in ie8 and they work unlike the previous version

to test my branch locally:
- clone my repo to your local Sites folder: `git clone git@github.com:cfarm/api-1.git`
- check out this branch: `git checkout ccdb-bug-fixes`
- run it locally: `bundle exec jekyll serve --watch --baseurl ''`
- view in browser: [http://localhost:4000/ccdb/](http://localhost:4000/ccdb/)
- compare to previous version here, especially release notes copy: http://cfarm.github.io/api-1/ccdb/data.html
## Review
- @huetingj - check release notes content + all copy + icons
- @vczosek - code review
## Screenshots

![screen shot 2016-01-13 at 1 40 31 pm](https://cloud.githubusercontent.com/assets/702526/12304006/61f82746-b9fb-11e5-99d0-a7d4a2b41408.png)

![screen shot 2016-01-13 at 1 40 57 pm](https://cloud.githubusercontent.com/assets/702526/12304009/6788d1d8-b9fb-11e5-93f5-4c2d04e9f81b.png)
![screen shot 2016-01-13 at 1 41 06 pm](https://cloud.githubusercontent.com/assets/702526/12304013/6b4d9880-b9fb-11e5-85f9-161f9af5395f.png)
## Checklist
